components/brave_wallet/browser/solana_instruction_builder.cc

```diff
@@ -125,6 +125,50 @@ absl::optional<SolanaInstruction> Transfer(
 
 }  // namespace spl_token_program
 
+namespace spl_associated_token_account_program {
+
+// Create an associated token account for the given wallet address and token
+// mint.
+// Account references:
+// 0. Funding account (must be a system account) [signer, writeable].
+// 1. Associated token account address to be created [non-signer, writable].
+// 2. Wallet address for the new associated token account [non-signer,
+//    readonly].
+// 3. The token mint for the new associated token account [non-signer,
+//    readonly].
+// 4. System program [non-signer, readonly].
+// 5. SPL Token program [non-signer, readonly].
+// 6. Rent sysvar [non-signer, readonly].
+// Ref:
+// https://docs.rs/spl-associated-token-account/1.0.3/spl_associated_token_account/fn.create_associated_token_account.html
+//
+// TODO(jocelyn): Update this to match
+// instruction::create_associated_token_account in spl-associated-token-account
```

Comment:
Is there an issue for this? to address the `Rent sysvar` removal

Reply:
Created https://github.com/brave/brave-browser/issues/21824, thanks.